src/bitcoin-tx.cpp

```diff
@@ -766,9 +771,7 @@ static std::string readStdin()
     if (ferror(stdin))
         throw std::runtime_error("error reading stdin");
 
-    boost::algorithm::trim_right(ret);
-
-    return ret;
+    return TrimRight(ret);
```

Comment:
I think `TrimString(…)` is okay to use here. That would remove the need to add and thus maintain `TrimRight(…)`.

Reply:
I'd be happy to do that but went with this initially as less likely to introduce behavior changes and perhaps easier to review.

Reply:
Replaced. 👍